cost/azure/reserved_instances/recommendations/azure_reserved_instance_recommendations.pt

```diff
@@ -143,13 +190,23 @@ script "js_ri_cleanup", type: "javascript" do
   EOS
 end
 
+###############################################################################
+# Policy
+###############################################################################
+
 policy "azure_ri_recommendations" do
   validate_each $ds_ri_normalization do
     summary_template "(Enrollment ID: {{parameters.param_enrollment}}): {{ len data }} Azure Reserved Instances Purchase Recommendations"
     escalate $email
     check lt(to_n(val(item,"netSavings")),$param_savings_threshold)
     export do
       resource_level true
+      field "subscriptionID" do
+        label "Subscription ID"
+      end
+      field "subscriptionName" do
+        label "Subscription Name"
+      end
```

Comment:
Should we consider calling these `accountID` and `accountName`, to be in line with the [standard fields](https://flexera.atlassian.net/wiki/spaces/PROD/pages/1712000589/Standard+Fields+for+Policy+Incidents) we'll scrape for recommendations?